pygen/pygen_src/isa/riscv_cov_instr.py

```diff
@@ -175,22 +175,21 @@ def pre_sample(self):
         self.rd_special_value = self.get_operand_special_value(self.rd_value)
         self.rs2_special_value = self.get_operand_special_value(self.rs2_value)
         self.rs3_special_value = self.get_operand_special_value(self.rs3_value)
-        if (self.format != riscv_instr_format_t.R_FORMAT and
-                self.format != riscv_instr_format_t.CR_FORMAT):
+        if (self.format.name != "R_FORMAT" and
```

Comment:
nit: maybe switch to `if (self.format.name not in ["R_FORMAT", "CR_FORMAT"]):` ?

Reply:
yeah sure, I missed this one after changed it to X.name. Will fix it.